### DISCUSSION TITLE
Does CustomMonoGraphQL support custom filter such as error handling and JWT?

### DISCUSSION THREAD

**Discussion by wyf0926** [Netflix/dgs-framework#1514] [General]
Hi there! Currently, we have a MonoGraphQLClient with custom error handling filter. Now we want to add one more JWT filter, whose token is generated based on secret key and the request body. However, after doing some research online, I found it really hard to access to the request body for a reactive MonoGraphQLClient. If we change it to the CustomGraphQLClient, I am not sure if it is still possible to have our own error handling filter and some other codecs config. Any advice and insights would be appreciated. Thank you!

**Comment by srinivasankavitha** [comment]
The GraphQLClient provided by the framework is a simple wrapper that also offers some convenience methods for processing the graphql response. There is no support for additional functionality beyond what the standard Webclient or your custom client offers.
For reference, https://netflix.github.io/dgs/advanced/java-client/
